packages/colorpickers/demo/colorPicker.stories.mdx

```diff
@@ -0,0 +1,46 @@
+import { Meta, ArgsTable, Canvas, Story } from '@storybook/addon-docs';
+import { useArgs } from '@storybook/client-api';
+import { PALETTE } from '@zendeskgarden/react-theming';
+import { Colorpicker } from '@zendeskgarden/react-colorpickers';
+
+<Meta
+  title="Packages/Colorpickers/Colorpicker"
+  component={Colorpicker}
+  parameters={{
+    design: {
+      allowFullscreen: true,
+      type: 'figma',
+      url: 'https://www.figma.com/file/6g87L4FdKZTA3knt3Rsfdx/Garden?node-id=2178%3A344'
+    }
+  }}
+/>
+
+# API
+
+<ArgsTable />
+
+# Demo
+
+## Uncontrolled
+
+<Canvas>
+  <Story name="Uncontrolled" argTypes={{ color: { control: false } }}>
```

Comment:
I noticed that setting `defaultColor` for the uncontrolled demo causes an error because Storybook passes `{}`. Should this knob be disabled (or maybe have a default set)?

<img width="1271" alt="Screen Shot 2022-01-04 at 2 21 10 PM" src="https://user-images.githubusercontent.com/1811365/148132439-5cbcbcaa-3c51-401b-a9ce-601f9f97f5a2.png">

Reply:
Yeah. I ended up leaving this in place, because if you enter a valid value such as `"#888"`, switch to another story, and then come back to the uncontrolled color picker – the `defaultValue` sticks as expected.

Reply:
This could present an opportunity to make the component a bit more robust 🤷 rather than taking away a control that manifests a component error.